octue/__init__.py

```diff
@@ -6,6 +6,8 @@
 
 __all__ = ("Runner",)
 
+
+DEFAULT_OCTUE_SERVICES_NAMESPACE = "octue.services"
```

Comment:
@thclark can you think of a name for this other than "namespace"? It's the name of the topic (or with a non-pub/sub backend, maybe the "workspace"?) that all services in a project/workspace emit and consume events to/from. I'd like it to not be specific to pub/sub so it translates to other future event-driven back ends.

Reply:
I'm also thinking that maybe we shouldn't make this configurable - maybe it should always be `octue.services`

Reply:
I can't see a use case for changing it that wouldn't break everything, happy with it being a constant. It could be `PREFIX` rather than `NAMESPACE`. Agree that calling it namespace conflicts with what namespaces are in the registry.